src/posts/sota.md

```diff
@@ -14,17 +14,16 @@ _Last update: 2024-09-16_
 - **Working memory**: An average human can remember about 7 items (such as numbers) at a time. Gemini 1.5 Pro [can read and remember 99% of 7 million words](https://blog.google/technology/ai/google-gemini-next-generation-model-february-2024/#sundar-note).
 - **Reading speed**: A model like Gemini 1.5 Pro can read an entire book in 30 seconds. It can learn an entirely new language and translate texts in half a minute.
-- **Thinking speed**: AI models can write at speeds far surpassing any human, writing entire computer programmes in seconds.
+- **Writing speed**: AI models can write at speeds far surpassing any human, writing entire computer programmes in seconds.
 - **Amount of knowledge**: Modern LLMs know far more than any human, its knowledge spanning virtually every domain. There is no human whose knowledge breadth comes close.
-- **Storage efficiency**: GPT-4 has about [1.7 trillion parameters](https://the-decoder.com/gpt-4-architecture-datasets-costs-and-more-leaked/) (neuron connections), whereas humans have [about 100 to 1000 times the number of synapses](https://www.jax.org/news-and-insights/jax-blog/2018/December/600-trillion-synapses-and-alzheimers-disease) (neuron connections). However, GPT-4 knows thousands of times more, storing more information in a smaller amount of parameters.
-
+  
 ## Better than most humans
 
-- **Language**: The best language models can translate virtually all languages fluently, have superhuman vocabulary and can write in many different styles. In December 2023, an AI-written novel won an award at a [science fiction national competition](https://www.scmp.com/news/china/science/article/3245725/chinese-professor-used-ai-write-science-fiction-novel-then-it-won-national-award?campaign=3245725&module=perpetual_scroll_0&pgtype=article). The professor who used the AI crafted the narrative from a draft of 43,000 characters generated in just three hours with 66 prompts.
-- **Reasoning**: o1 [correctly answers 78%](https://openai.com/index/learning-to-reason-with-llms/) of GPQA diamond questions, outperforming human domain experts (PhDs) who only get 69.7%.
+- **Writing**: In December 2023, an AI-written novel won an award at a [science fiction national competition](https://www.scmp.com/news/china/science/article/3245725/chinese-professor-used-ai-write-science-fiction-novel-then-it-won-national-award?campaign=3245725&module=perpetual_scroll_0&pgtype=article). The professor who used the AI crafted the narrative from a draft of 43,000 characters generated in just three hours with 66 prompts. The best language models have superhuman vocabulary and can write in many different styles.
+- **Translating**: And they can respond and translate to all major languages fluently.
```

Comment:
Removing the "reasoning" section is dubious IMO. It's an important skill that we need to list here.